.circleci/config.yml

```diff
@@ -226,6 +269,12 @@ workflows:
       - test-download-redmine
       - test-download-redmine-latest
       - test-download-redmine-trunk
+      - test-download-redmine-supported-max-version:
+          context:
+            - lychee-ci-environment
+      - test-download-redmine-supported-min-version:
+          context:
+            - lychee-ci-environment
```

Comment:
IMHO:

Forked repositories must set `lychee-ci-environment` context to test success. :eyes: But these tests are fully described our currently problem. :+1: :+1: :+1:

Another solution: set job environment to `test-download-redmine-supported-max-version` and `test-download-redmine-supported-min-version`.